Mathlib/Data/List/BigOperators/Basic.lean

```diff
@@ -447,6 +447,15 @@ theorem prod_drop_succ :
 #align list.prod_drop_succ List.prod_drop_succ
 #align list.sum_drop_succ List.sum_drop_succ
 
+/-- Cancellation of a telescoping product. -/
+@[to_additive "Cancellation of a telescoping product"]
+theorem prod_map_range_cancel (n : ℕ) (f : ℕ → G) :
```

Comment:
This seems very similar to `Finset.prod_range_div`. Do you think you can use that instead?

Reply:
`Finset.prod_range_div` assume `CommGroup`, but I suppose I could change this lemma to use `/`.

Reply:
Well actually, this lemma isn't quite the right way around to use `/`.

Reply:
I've shuffled things around to use `/`, but of course I still can't use `Finset.prod_range_div` directly.